app/src/main/java/io/stormbird/wallet/service/KeyService.java

```diff
@@ -528,99 +504,91 @@ private void importHDKey()
     {
         //first recover the seed phrase from non-authlocked key. This removes the need to keep the seed phrase as a member on the heap
```

Comment:
I didn't know that. I had thought that stack/heap worked the same in Java as in C++ but I suppose there's no reason why it would. Doing it this way does make the code slightly cleaner and ensures if something goes wrong while getting authentication the key will still be there.